src/native_background.ts

```diff
@@ -161,16 +224,55 @@ export async function mkdir(dir: string, exist_ok: boolean) {
 export async function temp(content: string) {
     return sendNativeMsg("temp", { content })
 }
+
 export async function run(command: string) {
     let msg = await sendNativeMsg("run", { command })
     logger.info(msg)
     return msg
 }
 
+export async function getenv(variable: string) {
+    let v = await getNativeMessengerVersion()
+    if (!nativegate("0.1.2", false)) {
```

Comment:
If you just call nativegate(..., true), it will handle telling the user that they need to update.

Is there a specific reason that you wanted to throw an error?

Reply:
I thought this function might end up being used somewhere else and users might not want the warning. I could have getenv accept a second argument and pass it to nativegate though, would you prefer that?

Reply:
> I thought this function might end up being used somewhere else and users might not want the warning. 

That's reasonable. Just be aware that anything you throw will eventually pop up on the command line if it isn't caught.

> I could have getenv accept a second argument and pass it to nativegate though, would you prefer that?

I don't understand what you're suggesting.

Reply:
I meant turning `getenv(variable: string)` into `getenv(variable: string, interactive: boolean)` and turning `nativegate("0.1.2", false)` into `nativegate("0.1.2", interactive)`.

Reply:
Ah, no, don't worry about it. Just leave it as it is :).